### PR TITLE
Tweak dxvk patches & build local & action

### PR DESCRIPTION
**Pull request by Gcenx**

### CODE REVIEW

**Comment by Gcenx** [comment]
@dasmy any thoughts on these changes?

File: build_local.command [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cd -- "$(dirname "$BASH_SOURCE")"
+./build_local.sh
```
Comment (dasmy) [inline build_local.command:3]: Why did you add the .command file if it only call the .sh file anyway?

File: build_local.sh [hunk 2]
```diff
@@ -113,85 +101,41 @@ if [[ ${CROSS_OVER_VERSION} == 20.* ]]; then
     pushd sources/wine
     patch -p1 < ${GITHUB_WORKSPACE}/msvcrt-sincos.patch
     popd
-
-    echo Patch DXVK
-    patch sources/dxvk/src/util/rc/util_rc_ptr.h < dxvk_util_rc_ptr.patch
 fi
 
-############ Build LLVM / Clang ##############
```
Comment (dasmy) [inline build_local.sh:121]: Why don't you build LLVM and Clang any more?

**Review by dasmy — COMMENTED** [review]
In general, things look OK (didn't run locally, though).

Should we possibly consider re-adding a GitHub action that explicitly runs build_local.sh to make sure it always works as expected?

File: build_local.command (hunk 1 above)
Comment (Gcenx) [reply build_local.command:3]: A command file can be launched directly from Finder using the defaults.

File: build_local.sh (hunk 2 above)
Comment (Gcenx) [reply build_local.sh:121]: That was replaced by using cx-llvm

**Comment by Gcenx** [comment]
> In general, things look OK (didn't run locally, though).
> 
> Should we possibly consider re-adding a GitHub action that explicitly runs build_local.sh to make sure it always works as expected?

I've ran this on my m1 mini and it completed without issue. Not sure about enabling an action for build_local.sh again though.

File: build_local.sh (hunk 2 above)
Comment (dasmy) [reply build_local.sh:121]: Oh, I did not spot the cx-llvm. This is pretty nice.

**Comment by dasmy** [comment]
> > In general, things look OK (didn't run locally, though).
> > Should we possibly consider re-adding a GitHub action that explicitly runs build_local.sh to make sure it always works as expected?
> 
> I've ran this on my m1 mini and it completed without issue. Not sure about enabling an action for build_local.sh again though.

Hmm... - In activating a build action I would see the advantage that we make sure it is kept in a working state with upcoming changes. Also, with removal of the clang/llvm build, it should be a bit quicker...

What would be reasons not to activate it? - I remember it ran almost 6hours when we deactivated it because nobody wanted to wait for it anyway...

Maybe we only activate it the action for `build_local.sh` only in main but not in PRs?

**Comment by Gcenx** [comment]
> Hmm... - In activating a build action I would see the advantage that we make sure it is kept in a working state with upcoming changes. Also, with removal of the clang/llvm build, it should be a bit quicker...
>
> What would be reasons not to activate it? - I remember it ran almost 6hours when we deactivated it because nobody wanted to wait for it anyway...

I believe the only real reason we’d disabled this was due to the time but side stepping that by using cx-llvm resolves a large chunk on this.

> Maybe we only activate it the action for `build_local.sh` only in main but not in PRs?

Maybe.

**Comment by Gcenx** [comment]
Actually for the configuration of build_local.sh it should be PR only and only ran if the other actions all passed as there’s no point running if the others failed

**Comment by dasmy** [comment]
With 8400349a4c5b853b749d1c964792cc6f18caf1b6 I tried to achieve that the build_local workflow runs after successful completion of build_monolithic, but it just didn't.

As usual, the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) is not very clear, but I assume the following is the reason:
`Note: This event will only trigger a workflow run if the workflow file is on the default branch.
`
Thus I pushed that change to main to see if it works.

**Comment by dasmy** [comment]
> With [8400349](https://github.com/GabLeRoux/macos-crossover-wine-cloud-builder/commit/8400349a4c5b853b749d1c964792cc6f18caf1b6) I tried to achieve that the build_local workflow runs after successful completion of build_monolithic, but it just didn't.
> 
> As usual, the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) is not very clear, but I assume the following is the reason: `Note: This event will only trigger a workflow run if the workflow file is on the default branch. ` Thus I pushed that change to main to see if it works.

*grml*
As the follow-up workflow must reside in the default branch (i.e. main), it also runs from that version instead of the PR branch. Clearly, this is useless. Thus, we can only add a job to the original workflow, which is no problem in general, just a bit ugly from the aesthetics perspective. :-(
